### ISSUE TITLE
Lists of selectors with `force-element-nesting`?

### ISSUE THREAD

**Issue by rschamp** [sasstools/sass-lint#575]
The default rules will warn about `force-element-nesting` if you have something like this because `.subclass` should be nested in `.class2`:
```scss
.class1,
.class2 .subclass {
// class `.subclass` should be nested within its parent class  (force-element-nesting)
  ...
}
```
How does one do that without duplicating the rules for both of these selectors?  Nesting `.subclass` in that block wouldn't work because it would apply the rules to `.class1 .subclass`.

The `force-element-nesting` rule should ignore nested rules within lists of selectors, or maybe there should be an option to allow it?

**Comment by nmcclay** [comment]
I also got bit by this limitation today.  In our case we were doing something like:
```
.btn,
button.btn,
a.btn {
// button styling
};
```
Obviously I could write a mixin or something to declare these individually, but this seems fairly reasonable to me.  I'd like to use this rule for preventing particularly egregious single line selectors, perhaps adding an optional depth parameter for how many elements it takes to violate the rule?

**Comment by bgriffith** [comment]
Thanks for flagging this up. I agree the logic behind this rule is currently a little too simple at the moment. I'll tag as an enhancement.

**Comment by ajithr** [comment]
I also faced the slimier warning when using below codes. 

```
.first.second {      
  &.wrap,
  &.wrap .title > div,
  &.text .title {
    // style
  }
}
```
is it also related to the same enhancement?

**Comment by chrismbarr** [comment]
Same here.  I'm needing to define CSS animations from Angular

```scss
#container {
	&.ng-enter,
	&.ng-leave {
		transition: transform 250ms, opacity 250ms;
	}

	&.ng-enter,
	&.ng-leave.ng-leave-active {
		opacity: 0;
		transform: translate3d(0, 300px, 0);
	}

	&.ng-leave,
	&.ng-enter.ng-enter-active {
		opacity: 1;
		transform: translate3d(0, 0, 0);
	}
}
```

Right now the only solution seems to be to do this...

```scss
#content {
	&.ng-enter,
	&.ng-leave {
		transition: transform 250ms, opacity 250ms;
	}

	&.ng-enter{
		opacity: 0;
		transform: translate3d(0, 300px, 0);

		&.ng-enter-active {
			opacity: 1;
			transform: translate3d(0, 0, 0);
		}
	}
			
	&.ng-leave {
		opacity: 1;
		transform: translate3d(0, 0, 0);

		&.ng-leave-active {
			opacity: 0;
			transform: translate3d(0, 300px, 0);
		}
	}
}
```

Now, it's not the worst... but the 1st version just seems easier to understand.  I hate having to re-write perfectly good code just to satisfy a rule.

**Comment by jrmykolyn** [comment]
Would love to see some additional options around this issue. Any movement?
